src/Handler.php

```diff
@@ -132,21 +140,25 @@ protected function getExcludes() {
    * Respects 'omit-defaults', and either includes or does not include the
    * default values, as requested.
    *
-   * @return array
+   * @return array[]
```

Comment:
Either array if it's a mixed array or objectType[] if the type of each element in the array is the same (eg: string[]).
Unless this returns an array of arrays, in which case this is fine, although I think just array would suffice in that case to avoid confusion?